uPy/wifi_connect.py

```diff
@@ -79,6 +80,12 @@ def station_connected(station: WLAN, host: String, gdt: GDT, wifiLogger: Logger)
             print("Splashpage Length [{}]".format(len(splashpage)))
             
             print("Splashpage Breaking...")
+
+            # break with form resubmission
+            forms = get_forms(splashpage)
```

Comment:
can you comment these lines out like with the a tag breaking method so that we're not automatically using this